documentation/meta/maintenance/elasticsearch_cluster.md

```diff
@@ -376,8 +376,8 @@ best to pause it for performance reasons.
 1. Configure the created instance by running our ansible playbook to sync
    Elasticsearch nodes. Supply the correct environment name to the command and
    pass the DNS record from the previous step to the limit flag like so:
-   `just ansible/playbook <env> elasticsearch/sync_config.yml -e apply=true -l <public_ipv4_dns>`
-1. Use `just jh es <env>` to connect to the cluster, and send
+   `./ov just ansible/playbook <env> elasticsearch/sync_config.yml -e apply=true -l <public_ipv4_dns>`
+1. Use `./ov just jh es <env>` to connect to the cluster, and send
```

Comment:
Similar to the above, these also reference a different repo, and so they'll just be bare:
```suggestion
   `just ansible/playbook <env> elasticsearch/sync_config.yml -e apply=true -l <public_ipv4_dns>`
1. Use `just jh es <env>` to connect to the cluster, and send
```